Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/azure-policy-deploy.yaml

```diff
@@ -31,32 +31,31 @@ spec:
       - name: azure-policy
         image: {{ .Values.azurepolicy.image.name }}:{{ .Values.azurepolicy.image.tag }}
         resources:
-{{ toYaml .Values.azurepolicy.resources.deployment | indent 9 }}
```

Comment:
Why are we hardcoding these instead of reading from the values file?

Reply:
As of now I do not see a need for customizing/parametrizing Azure policy resources like Gatekeeper. In future if needed, we can parameterize this.

Reply:
Sounds good